.chloggen/spanmetricsconnector_disable_options.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: spanmetricsconnector
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Added disabling options for histogram and sum metrics. In some cases users wants to optionally disable part of metrics cause they can generate too many data or just not needed."
```

Comment:
```suggestion
note: "Added disabling options for histogram and sum metrics. In some cases users want to optionally disable part of metrics because they generate too much data or are not needed."
```